easybuild/easyblocks/l/lammps.py

```diff
@@ -93,16 +93,18 @@ class EB_LAMMPS(CMakeMake):
     @staticmethod
     def extra_options(**kwargs):
         """Custom easyconfig parameters for LAMMPS"""
-
-        extra_vars = {
+        extra_vars = CMakeMake.extra_options()
+        extra_vars.update({
             # see https://developer.nvidia.com/cuda-gpus
             'cuda_compute_capabilities': [[], "List of CUDA compute capabilities to build with", CUSTOM],
             'general_packages': [None, "List of general packages without `PKG_` prefix.", MANDATORY],
             'kokkos': [True, "Enable kokkos build.", CUSTOM],
             'kokkos_arch': [None, "Set kokkos processor arch manually, if auto-detection doesn't work.", CUSTOM],
             'user_packages': [None, "List user packages without `PKG_USER-` prefix.", MANDATORY],
-        }
-        return CMakeMake.extra_options(extra_vars)
+        })
+        extra_vars['separate_build_dir'][0] = False
+        extra_vars['build_shared_libs'][0] = True
```

Comment:
Maybe we should do this in the CMake EB? Add it to `options` instead of `configopts` so it is added before `configopts` and hence can be overwritten by it?

Reply:
I'm strongly in favor of avoiding to include the same options multiple times in the `cmake` command, that'll only lead to confusion...